MAUI/Busy-Indicator/Getting-Started.md

```diff
@@ -161,5 +161,5 @@ The following gif image illustrates the result of the above code.
 
 ![Getting Started](Images/GettingStarted/CircularMaterial.gif)
 
-You can find the complete getting started sample of .NET MAUI Busy Indicator from this [link.](https://github.com/SyncfusionExamples/maui-busyindicator-samples)
+N> You can refer to our [.NET MAUI Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Busy Indicator Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator/SampleBrowser.Maui.BusyIndicator/Samples/BusyIndicator) that shows you how to render the BusyIndicator in .NET MAUI.
```

Comment:
Busy Indicator in .NET MAUI.
![image](https://github.com/syncfusion-content/maui-docs/assets/95272441/f3411264-8366-4298-9d7d-de14a1bdf7d0)

Reply:
The issue is resolved.